simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/HotspotMetricCollectorsConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package io.prometheus.client.spring;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.prometheus.client.Summary;
+import io.prometheus.client.hotspot.ClassLoadingExports;
+import io.prometheus.client.hotspot.GarbageCollectorExports;
+import io.prometheus.client.hotspot.MemoryPoolsExports;
+import io.prometheus.client.hotspot.StandardExports;
+import io.prometheus.client.hotspot.ThreadExports;
+
+/**
+ * A {@link Configuration} class that imports the Prometheus
+ * JVM management metrics Collectors from the <code>simpleclient_hotspot</code> package.
+ *
+ * @author pidster
+ */
+@Configuration
+public class HotspotMetricCollectorsConfiguration {
```

Comment:
All you should need to do is ensure DefaultExports gets initialised at least once. I'm not going to maintain that list in two places.